directives/questionnaire.py

```diff
@@ -200,6 +204,25 @@ def run(self):
             if 'url' in data:
                 data['url'] = data['url'].format(key=name)
 
+        source, line = self.state_machine.get_source_and_line(self.lineno)
+        if 'reveal-submission-feedback' in self.options:
+            data['reveal_submission_feedback'] = parse_reveal_rule(
+                self.options['reveal-submission-feedback'],
+                source,
+                line,
+                'reveal-submission-feedback',
+            )
+        if 'reveal-model-solutions' in self.options:
+            data['reveal_model_solutions'] = parse_reveal_rule(
+                self.options['reveal-model-solutions'],
+                source,
+                line,
+                'reveal-model-solutions',
+            )
+
+        if 'grading-mode' in self.options:
+            data['grading_mode'] = self.options['grading-mode']
```

Comment:
Like I commented in submit.py, these lines shall be moved before the `override` part. Before the line `if not 'no-override' in self.options and category in override:`

Reply:
Moved the lines